components/ChartWrapper.js

```diff
@@ -12,6 +12,9 @@ export const ChartWrapper = styled.div`
       vertical-align: middle;
     }
   }
+  .apexcharts-toolbar {
+    z-index: 0;
+  }
```

Comment:
```suggestion
```

One small suggestion is to remove this block. For me this works even without this block. Can you double check as well? 🤔

Reply:
the issue was happening on 2 graphs
<img width="1027" alt="Screenshot 2023-01-04 at 10 34 42 AM" src="https://user-images.githubusercontent.com/29824008/210525786-0d259b75-158c-43ce-9f1b-f2f4a3044fe5.png">

When I remove it the chart for Total money managed has the bug.

Reply:
Ops. I missed this one.... All good. 👍🏽